src/components/FormInput.vue

```diff
@@ -11,14 +11,7 @@
       :class="classList"
       v-on:blur="formatFloatValue()"
     >
-      <template v-if="validator && validator.errorCount">
-        <div class="invalid-feedback" v-for="(errors, index) in validator.errors.all()" :key="index">
-          <div v-for="(error, subIndex) in errors" :key="subIndex">
-            {{error}}
-          </div>
-        </div>
-      </template>
-      <div class="invalid-feedback" v-if="error">{{error}}</div>
+    <display-errors v-if="error || (validator && validator.errorCount)" :name="name" :error="error" :validator="validator"/>
```

Comment:
I don't think the `v-if` on the `display-errors` component is needed, since the template of that component already contains a `v-if`  (`v-if="displayErrors"`) that seems to accomplish the same thing.